geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/EnvironmentCommand.scala

```diff
@@ -44,26 +94,45 @@ class EnvironmentCommand(parent: JCommander) extends Command(parent) with LazyLo
   def listSfts(names: List[String] = List.empty): Unit = {
     val all = SimpleFeatureTypeLoader.sfts
     val filtered = if (names.isEmpty) all else names.flatMap(n => all.find(_.getTypeName == n))
-    println("Simple Feature Types:")
+    println("\nSimple Feature Types:")
     if (filtered.isEmpty) {
       println("\tNone available")
     } else {
-      filtered.map(s => s"\t${s.getTypeName} = ${SimpleFeatureTypes.encodeType(s)}").foreach(println)
+      //filtered.sortBy(_.getTypeName).map(s => s"\t${s.getTypeName} = ${SimpleFeatureTypes.encodeType(s)}").foreach(println)
+      params.format.toLowerCase match {
+        case "typesafe" =>
+          filtered.sortBy(_.getTypeName).map(s => s"\t${s.getTypeName} = ${SimpleFeatureTypes.toConfigString(s, !params.excludeUserData, params.concise)}").foreach(println)
+        case "spec" =>
+          filtered.sortBy(_.getTypeName).map(s => s"\t${s.getTypeName} = ${SimpleFeatureTypes.toConfigString(s, !params.excludeUserData)}").foreach(println)
```

Comment:
this may be a nice place to use a first class function...like
```
val encodeFn = params.format.toLowerCase match {
   case "ts" => (sft) => ${SimpleFeatureTypes.toConfigString(s, !params.excludeUserData, params.concise)}"
   case "spec" => ....etc....
}
filtered.sortBy(_.getTypeName).map(s => s"\t${s.getTypeName} = ${fn(s)").foreach(println)
```
or something like that